shift/demo/templates.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+     Copyright 2019 Coop IT Easy
```

Comment:
```suggestion
     Copyright 2019-2023 Coop IT Easy SC
```

Reply:
done

Reply:
@remytms @victor-champonnois bumping the year is not necessary. see good blog article:

https://matija.suklje.name/how-and-why-to-properly-write-copyright-statements-in-your-code#why-not-bump-the-year-on-change

one less thing to worry about :)